legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -172,7 +172,7 @@ def search_businesses():
         draft_results = []
         for draft_dao in draft_query.all():
             draft = {
-                'identifier': draft_dao.temp_reg,  
+                'identifier': draft_dao.temp_reg,
```

Comment:
I think Vysakh is making changes here so I will rebase once that commit has gone through